File: app/assets/javascripts/models/Reservation.js
var app = app || {};

app.Reservation = Backbone.Model.extend({
  urlRoot: '/reservations',

  defaults: {
    user_id: 0,
    flight_id: 0,
    row: "0",
    col: "0"
  }
});

// create_table "reservations", force: :cascade do |t|
//   t.integer  "user_id"
//   t.integer  "flight_id"
//   t.string   "row"
//   t.string   "col"
//   t.datetime "created_at", null: false
//   t.datetime "updated_at", null: false
// end

File: app/models/reservation.rb
# == Schema Information
#
# Table name: reservations
#
#  id         :integer          not null, primary key
#  user_id    :integer
#  flight_id  :integer
#  row        :string
#  col        :string
#  created_at :datetime         not null
#  updated_at :datetime         not null
#

class Reservation < ActiveRecord::Base
  belongs_to :flight
  belongs_to :user
end
